mne/source_space.py

```diff
@@ -1182,6 +1182,54 @@ def vertex_to_mni(vertices, hemis, subject, subjects_dir=None, mode=None,
     return apply_trans(xfm['trans'], data)
 
 
+##############################################################################
+# Volume to MNI conversion
+
+
+@verbose
+def aseg_vertex_to_mni(vertices, subject, mri_head_t, subjects_dir=None,
+                       verbose=None):
+    """Convert the array of vertices for a hemisphere to MNI coordinates.
+
+    Parameters
+    ----------
+    vertices : n_vertices x 3 array of float
+        The  coordinates (in m) of the vertices in head coo system
+    subject : string
+        Name of the subject to load surfaces from.
+    mri_head_t: Transform
```

Comment:
You can just call this `trans`, we do this in user-facing functions

Reply:
done